usage-examples/editorial-commands.ly

```diff
@@ -0,0 +1,34 @@
+\version "2.19.42"
+
+\include "scholarly/package.ly"
+
+\setOption scholarly.editorial.addition #`(
```

Comment:
>>  This forces the end user to define the editorial functions, otherwise LilyPond will fail.
> Will this not be resolved once we use getChildOptionWithFallback?

Yes, the error. But not the requirement to explicitly define functions before being able to use them

> Quasi/un-quote is a small price to pay, but it does work. 

That's right. Let stuff this reservation pretty much to the back of the mind (i.e. don't discard it completely but agree it's not a show-stopper).

---

Let me try to explain better what I wanted to say (and what I originally had in mind).

```
music = {
  \criticalRemark
  \with{
    message = "A remark about adding the slur."
    apply = addition
} Slur a4( b c') d'
```
is indeed how the interface in the music document should look like. I definitely didn't want to suggest something like `apply = slurDashed`.

But inside the `annotate` function I would not like to directly invoke some actual override function like `\slurDashed` but add another layer of abstraction. The above annotation should trigger a function like `\editorialAddition`. This function (and its deletion/emendation or whatever sibling functions) is defined in another module of the ScholarLY package, so users can apply `\editorialAddition` independently from annotations. The lookup of what will concretely be done with the grob can then be implemented through the `\setOption` approach just like you already did. I would just add one more layer in the option path, like `scholarly.editorial.functions.addition`